assets/dev/scss/global/global.scss

```diff
@@ -10,6 +10,10 @@
 	display: none;
 }
 
+.elementor-visibility-hidden {
+	visibility: hidden;
```

Comment:
@louiswol94 because visibility hidden keeps the layout, while display none breakes it. I also tweaked slightly the template because I need to display the name of the edited component.